src/components/Chart/ChartBar.js

```diff
@@ -1,27 +1,37 @@
 // styles
-import styles from './ChartBar.module.css'
+import React from "react";
+import PropTypes from "prop-types";
+import styles from "./ChartBar.module.css";
 
-const ChartBar = props => {
-    // calculate bar fill height with variable that will help set CSS
-    let barFill = '0%'
+const ChartBar = ({ value, maxValue, label }) => {
+  // calculate bar fill height with variable that will help set CSS
+  const barFill = `${Math.round((value / maxValue) * 100)}%`;
 
-    // if max value from props is higher than zero convert to percentage and save as barFill
-    // also convert to string with % sign at end
-    if (props.maxValue > 0) {
-        barFill = Math.round((props.value / props.maxValue) * 100) + '%'
-    }
+  return (
+    <div className={styles["chart-bar"]}>
+      <div className={styles["chart-bar-inner"]}>
+        {/* dynamically adjust height with barFill variable in style react prop */}
+        {/* value is a javascript object */}
+        <div
+          className={styles["chart-bar-fill"]}
+          style={{ height: barFill }}
+        ></div>
+      </div>
+      {/* put label from props in this one */}
+      <div className={styles["chart-bar-label"]}>{label}</div>
+    </div>
+  );
+};
 
-    return (
-        <div className={styles["chart-bar"]}>
-            <div className={styles["chart-bar-inner"]}>
-                {/* dynamically adjust height with barFill variable in style react prop */}
-                {/* value is a javascript object */}
-                <div className={styles["chart-bar-fill"]} style={{ height: barFill }} ></div>
-            </div>
-            {/* put label from props in this one */}
-            <div className={styles["chart-bar-label"]}>{props.label}</div>
-        </div>
-    )
-}
+ChartBar.defaultProps = {
+  value: 0,
+  maxValue: 1,
+};
 
-export default ChartBar
\ No newline at end of file
+ChartBar.propTypes = {
+  value: PropTypes.number,
+  maxValue: PropTypes.number,
+  label: PropTypes.string.isRequired,
+};

```

Comment:
Defining Prop types will really amp up your codebase- `prop-types` is built into react, and really helps for debugging if something doesn't behave the way it should. Additionally you can set defaults (which I did here to cut down on your conditionals but can also be handy to set `undefined` or anything else you want